TestAccAWSS3Bucket_Logging Panic

```
Test ended in panic.

------- Stdout: -------
=== RUN   TestAccAWSS3Bucket_Logging
--- FAIL: TestAccAWSS3Bucket_Logging (5.95s)

------- Stderr: -------
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
    panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x25a19f8]

goroutine 18 [running]:
testing.tRunner.func1(0xc4205092c0)
    /usr/local/go/src/testing/testing.go:711 +0x2d2
panic(0x2bedc20, 0x531e2b0)
    /usr/local/go/src/runtime/panic.go:491 +0x283
github.com/terraform-providers/terraform-provider-aws/aws.testAccCheckAWSS3BucketLogging.func1(0xc4207b7260, 0x0, 0x0)
    /opt/teamcity-agent/work/6025502d125bbe5e/src/github.com/terraform-providers/terraform-provider-aws/aws/resource_aws_s3_bucket_test.go:1194 +0x278
github.com/terraform-providers/terraform-provider-aws/vendor/github.com/hashicorp/terraform/helper/resource.ComposeTestCheckFunc.func1(0xc4207b7260, 0xc4207b7260, 0x0)
    /opt/teamcity-agent/work/6025502d125bbe5e/src/github.com/terraform-providers/terraform-provider-aws/vendor/github.com/hashicorp/terraform/helper/resource/testing.go:815 +0x88
github.com/terraform-providers/terraform-provider-aws/vendor/github.com/hashicorp/terraform/helper/resource.testStep(0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
    /opt/teamcity-agent/work/6025502d125bbe5e/src/github.com/terraform-providers/terraform-provider-aws/vendor/github.com/hashicorp/terraform/helper/resource/testing_config.go:81 +0xddc
github.com/terraform-providers/terraform-provider-aws/vendor/github.com/hashicorp/terraform/helper/resource.testStepConfig(0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
    /opt/teamcity-agent/work/6025502d125bbe5e/src/github.com/terraform-providers/terraform-provider-aws/vendor/github.com/hashicorp/terraform/helper/resource/testing_config.go:17 +0x98
github.com/terraform-providers/terraform-provider-aws/vendor/github.com/hashicorp/terraform/helper/resource.Test(0x52bfa80, 0xc4205092c0, 0x0, 0xc4200f16c0, 0xc4204e8a20, 0x0, 0x0, 0x3390b38, 0xc420157080, 0x1, ...)
    /opt/teamcity-agent/work/6025502d125bbe5e/src/github.com/terraform-providers/terraform-provider-aws/vendor/github.com/hashicorp/terraform/helper/resource/testing.go:492 +0x1439
github.com/terraform-providers/terraform-provider-aws/aws.TestAccAWSS3Bucket_Logging(0xc4205092c0)
    /opt/teamcity-agent/work/6025502d125bbe5e/src/github.com/terraform-providers/terraform-provider-aws/aws/resource_aws_s3_bucket_test.go:594 +0x291
testing.tRunner(0xc4205092c0, 0x338bbd0)
    /usr/local/go/src/testing/testing.go:746 +0xd0
created by testing.(*T).Run
    /usr/local/go/src/testing/testing.go:789 +0x2de
```

--- a/aws/resource_aws_s3_bucket_test.go
+++ b/aws/resource_aws_s3_bucket_test.go
@@ -1189,6 +1189,10 @@ func testAccCheckAWSS3BucketLogging(n, b, p string) resource.TestCheckFunc {
 			return fmt.Errorf("GetBucketLogging error: %v", err)
 		}
 
+		if out.LoggingEnabled == nil {
+			return fmt.Errorf("logging not enabled for bucket: %s", rs.Primary.ID)
+		}
+
 		tb, _ := s.RootModule().Resources[b]
 
 		if v := out.LoggingEnabled.TargetBucket; v == nil {

